app/javascript/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+class App extends React.Component {
+  render () {
+    return (
+      <div>
+        <h1>Test React App</h1>
+        <p>Hello world!</p>
+      </div>
+    );
+  }
+}
+
+export default App
```

Comment:
```suggestion
import React, {Component} from 'react';

class App extends Component {
  render () {
    return (
      <div>
        <h1>Test React App</h1>
        <p>Hello world!</p>
      </div>
    );
  }
}

export default App;
```
Always put `;` at the end of each line.